### PR TITLE
Create a blog post for week 16, 2020

### PR DESCRIPTION
**Pull request by csomh**

### CODE REVIEW

File: content/posts/April-2020.md [hunk 1]
```diff
@@ -44,3 +44,25 @@ draft: false
...
+
+### packit
+
+* Job metadata field `dist-git-branch` is now marked as deprecated, to be
+  replaced by `dist_git_branches` to match the naming of other metadata fields
+  and to accept multiple branch names where Packit should work. ([#797] and
+  [#788].)
+* Two new job metadata fields, `scratch` and `branch`, were introduced. These
+  will be used by Packit when building in Koji becomes available. ([#796] and
+  [#800].)
```
Comment (csomh) [inline content/posts/April-2020.md:58]: @lachmanfrantisek is this correct?

Comment (lachmanfrantisek) [reply content/posts/April-2020.md:58]: - `scratch` will be used for koji builds.
- `branch` can be used for copr builds (not announced, not documented and not turned-on in the production GitHub app settings yet) and will be used for koji builds as well.

Comment (csomh) [reply content/posts/April-2020.md:58]: We discussed that it's better to drop this for now and announce them once they are available.

**Comment by csomh** [comment]
Rephrased a significant part, please take another look.

**Pull request merged**